diesel/src/mysql/connection/bind.rs

```diff
@@ -154,6 +154,11 @@ impl From<u32> for Flags {
     }
 }
 
+#[cfg(target_os = "linux")]
+static FALSE: ffi::my_bool = 0;
+#[cfg(any(target_os = "macos", target_os = "windows"))]
```

Comment:
If [Conditional compilation](https://doc.rust-lang.org/reference/conditional-compilation.html#conditional-compilation) supported `mysql_client_version` is better.

Reply:
I will try [cargo::rustc-cfg=KEY[="VALUE"]](https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-cfg)